tests/helper.js

```diff
@@ -217,14 +206,13 @@ module.exports = {
    * element : select element
    * index : index in the dropdown, 1 base.
    */
-  selectDropdownByNumber: (element, index, milliseconds) => {
-    element.findElements(by.tagName('option')).then(options => {
-      options[index].click();
-    }).catch(err => {
-      throw err;
-    });
+  selectDropdownByNumber: async (element, index, milliseconds) => {
```

Comment:
I went looking to see what milliseconds were being passed to this. When searching the repo for `selectDropdownByNumber`, it does not seem to be used anywhere.

Reply:
Yea, it's not used. Do you think we should remove it or leave it? It might be helpful someday ...

Reply:
Yeah let's leave it for now.